src/v2/Components/Artwork/__tests__/Details.jest.tsx

```diff
@@ -18,7 +19,9 @@ describe("Details", () => {
   ) => {
     return await renderRelayTree({
       Component: props => (
-        <DetailsFragmentContainer {...(props as any)} {...restProps} />
+        <ArtworkGridContextProvider isAuctionArtwork={true}>
```

Comment:
Note for truthy props, don't need `={true}`; `<ArtworkGridContextProvider isAuctionArtwork>` will suffice.

Reply:
Oh yes forgot about that